addons/l10n_fr_fec/wizard/account_fr_fec.py

```diff
@@ -365,6 +366,9 @@ def generate_fec(self):
         OFFSET %(offset)s
         '''
 
+        if self.exclude_zero:
+            rows_to_write = [row for row in rows_to_write if row[11] != '0,00' or row[12] != '0,00']
```

Comment:
Why don't we filter in SQL?
It would also allow filtering in only one place instead of 2

Reply:
Not if you implement it in `_get_where_query`

Reply:
@william-andre 
Yes, I did that when I was targeting 17.0 but when retargeted master, everything changed actually.
The module that I was in moved to l10n_fr_account and now the code depends on `_get_base_domain` to create domain list which is fine to add the condition there. but when we pass this list to `['table']._search` in order to generate the where statement. It fails because the `_search` doesn't have the ability to filter on a column that gets generated in the select statement e.g., 
```
replace(CASE WHEN account_move_line.debit = 0 THEN '0,00' ELSE to_char(account_move_line.debit, '000000000000000D99') END, '.', ',') AS Debit,
                replace(CASE WHEN account_move_line.credit = 0 THEN '0,00' ELSE to_char(account_move_line.credit, '000000000000000D99') END, '.', ',') AS Credit
```
NOTE:
Also, Sometimes the credit and debits not `account_move_line.debit` e.g.,
`replace(CASE WHEN sum(account_move_line.balance) <= 0 THEN '0,00' ELSE to_char(SUM(account_move_line.balance), '000000000000000D99') END, '.', ',') AS Debit,
                replace(CASE WHEN sum(account_move_line.balance) >= 0 THEN '0,00' ELSE to_char(-SUM(account_move_line.balance), '000000000000000D99') END, '.', ',') AS Credit`

Reply:
Added `_exclude_zero_debit_credit` to have the logic centerlized